server/lib/db/init-scripts/install-extensions.sql

```diff
@@ -0,0 +1,4 @@
+-- Sql script which installs necessary extensions for the shieldbattery server
```

Comment:
There's something I wanted to note about this file (which I somehow alluded to in the migration guide). Which is that it feels weird to even have it as it's not that useful. The way our docker service is running now is that it will run this script _only_ when the database is empty, which usually happens only when running the service the first time. And that means that if we want to install another extension once the database is already set up, it can't be put into this file, as it won't run again.

I feel like it might be more useful to have dedicated migrations for extension installing, as migrations will be run every time a service starts. I don't know if there are any downsides to installing extensions through migrations though. And also, even if add a migration to install these two extensions now, they would still need to be installed manually when setting up a server for the first time since our previous migrations depend on them.

Reply:
Can't use migrations for installing extensions because it needs superuser privileges. I tried many approaches, none of them really worked, so I just put it in the init scripts. If more extensions are ever need it, they can just be run one-off using `docker exec` prior to deploying that version (example in the init script itself). New extensions should be added to the init script so that brand new deploys get them by default.